Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Generator.cs

```diff
@@ -46,6 +46,14 @@ public void Execute(GeneratorExecutionContext context)
                     return;
                 }
 
+                // If there are no Lambda methods, return early
+                if (!receiver.LambdaMethods.Any())
+                {
+                    // If there are more than one startup class, report them as errors
```

Comment:
This comment doesn't match what is being done. Also do we need to report anything. The message for `LambdaFunctionNotFound` doesn't sound like just info. 

The tone make its sound like something is wrong and they must take action. In my case since it was showing up in the test project that has a dependency on the Lambda project they shouldn't see anything.

Reply:
I removed the info diag.